mousestyles/path_diversity/filter_path.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function, absolute_import, division
+
+
+def filter_paths(movement, paths, time_threshold):
+    r"""
+    Return a list object containing start and end indices
+    for a specific movement which the time spending on the
+    movement is larger than a time threshold
+

```

Comment:
Return a list object containing start and end indices for movements lasting longer than the specified time threshold

Might capture what the function does a little better? @jJasonWang